src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplTest.java

```diff
@@ -420,6 +422,57 @@ public void testGetCaseByUprn_caseSvcNotFoundResponse_noCachedCase() throws Exce
     verifyNewCase(addressFromAI, results.get(0));
   }
 
+  @Test
+  public void testGetCaseByUprn_caseSvcNotFoundResponse_noCachedCase_SPG() throws Exception {
+
+    AddressIndexAddressCompositeDTO addressFromAI =
+        FixtureHelper.loadClassFixtures(AddressIndexAddressCompositeDTO[].class).get(0);
+    addressFromAI.setCensusEstabType("Marina");
```

Comment:
I fell in love with a Marina once. Lovely girl.

Reply:
Changed to all lower case - "marina", as in a specially built harbour for a collection of small pleasure craft. Was actually just using mixed case but don't want to invoke fond memories.